src/backend/launcher.ts

```diff
@@ -243,6 +243,7 @@ async function prepareLaunch(
   let steamRuntime: string[] = []
   const shouldUseRuntime =
     gameSettings.useSteamRuntime &&
+    gameSettings.wineVersion.type === 'proton' &&
```

Comment:
```suggestion
    (!isNative && gameSettings.wineVersion.type === 'proton') &&
```

maybe?

because with the current code then you wouldn't be able to use it for native games which sometimes fixes issues